config/settings.yml

```diff
@@ -1,5 +1,5 @@
 library:
-  profile_url: 'https://library.stanford.edu/people/'
+  profile_url: 'https://library.stanford.edu/staff-directory'
```

Comment:
I don't think that's what this URL is; it used to point at the "people" pages on the library website.. e.g. https://library.stanford.edu/people/hanna-ahn.. but now they're not keyed by sunetid so I'm not sure how we could discover their existence?